DESCRIPTION

```diff
@@ -34,18 +34,14 @@ Imports:
     digest,
     downlit,
     fs (>= 1.3.0),
-    htmltools,
     httr (>= 1.4.2),
     magrittr,
     memoise,
-    openssl,
```

Comment:
hard dependency of httr and only used
* for building favicons;
* in a superseded function for deployment.